src/wallet/wallet.cpp

```diff
@@ -2627,6 +2671,38 @@ void CWallet::IncrementNoteWitnesses(
         bool performOrchardWalletUpdates)
 {
     LOCK(cs_wallet);
+
+    // Read the block from disk if we don't already have it.
+    const CBlock* pblock {pblockIn};
+    CBlock block;
+    if (!pblock) {
+        ReadBlockFromDisk(block, pindex, consensus);
```

Comment:
`ReadBlockFromDisk` returns a `bool` which is ignored here. Other calls to it do attempt to handle/report errors. I realize the code before this PR ignored errors from `ReadBlockFromDisk`.